ext/pgsql/tests/9999dropdb.phpt

```diff
@@ -3,17 +3,24 @@ PostgreSQL drop db
 --EXTENSIONS--
 pgsql
 --SKIPIF--
-<?php include("skipif.inc"); ?>
+<?php include("inc/skipif.inc"); ?>
 --FILE--
 <?php
 // drop test table
 
-include('config.inc');
+include('inc/config.inc');
+$table_name = "table_9999dropdb";
+$table_name_92 = "table_9999dropdb_92";
+$view_name = "view_9999dropdb";
 
 $db = pg_connect($conn_str);
-pg_query($db, "DROP VIEW {$view_name}");
-pg_query($db, "DROP TABLE ".$table_name);
-@pg_query($db, "DROP TABLE ".$table_name_92);
+pg_query($db, "CREATE TABLE {$table_name} (num int, str text, bin bytea)");
+pg_query($db, "CREATE TABLE {$table_name_92} (textary text[],  jsn json)");
+pg_query($db, "CREATE VIEW {$view_name} as SELECT * FROM {$table_name}");
+
+pg_query($db, "DROP VIEW IF EXISTS {$view_name}");
+pg_query($db, "DROP TABLE IF EXISTS {$table_name}");
+pg_query($db, "DROP TABLE IF EXISTS {$table_name_92}");
 
 echo "OK";
```

Comment:
This should be in a CLEAN section

Reply:
Fixed in https://github.com/php/php-src/pull/12608/commits/eadb74cb36cae1a0326cc984f176de95630033c1

Just to be safe, I dropped it in both the test and `--CLEAN--`, and removed `IF EXISTS` in the test.
Since the test name suggests that it is a `DROP` test, I thought that if it fails, it should be reported as a test failure.